gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceStubClassComposer.java

```diff
@@ -1248,4 +1249,56 @@ protected List<Statement> createTypeRegistry(Service service) {
                 .setValueExpr(typeRegistryBuilderExpr)
                 .build()));
   }
+
+  @Override
+  protected List<MethodDefinition> createClassMethods(
+      GapicContext context,
+      Service service,
+      TypeStore typeStore,
+      Map<String, VariableExpr> classMemberVarExprs,
+      Map<String, VariableExpr> callableClassMemberVarExprs,
+      Map<String, VariableExpr> protoMethodNameToDescriptorVarExprs,
+      List<Statement> classStatements) {
+    List<MethodDefinition> javaMethods = new ArrayList<>();
+    javaMethods.addAll(
+        super.createClassMethods(
+            context,
+            service,
+            typeStore,
+            classMemberVarExprs,
+            callableClassMemberVarExprs,
+            protoMethodNameToDescriptorVarExprs,
+            classStatements));
+    javaMethods.addAll(createInvalidClassMethods(service));
+    return javaMethods;
+  }
+
+  private List<MethodDefinition> createInvalidClassMethods(Service service) {
+    List<MethodDefinition> methodDefinitions = new ArrayList<>();
+    for (Method protoMethod : service.methods()) {
+      if (!protoMethod.isSupportedByTransport(getTransport())) {
+        String javaStyleProtoMethodName = JavaStyle.toLowerCamelCase(protoMethod.name());
+        String callableName =
+            String.format(CALLABLE_CLASS_MEMBER_PATTERN, javaStyleProtoMethodName);
+        methodDefinitions.add(
+            MethodDefinition.builder()
+                .setIsOverride(true)
+                .setScope(ScopeNode.PUBLIC)
+                .setName(callableName)
+                .setReturnType(getCallableType(protoMethod))
+                .setBody(
+                    Arrays.asList(
+                        ExprStatement.withExpr(
+                            ThrowExpr.builder()
+                                .setType(FIXED_TYPESTORE.get("UnsupportedOperationException"))
+                                .setMessageExpr(
+                                    String.format(
+                                        "Not implemented: %s(). %s transport is not implemented for this method yet",
```

Comment:
Now that we only have one error message, we may want to be a little more careful with the message. `transport is not implemented for this method yet` sounds like we didn't get a chance to implement it yet which is not the case sometimes(for BIDI and CLIENT streaming). I don't have a better message either, but let's give it a little more thought, maybe we can look at what the message is for other languages as well.

Reply:
Yeah, I'll try to think of some better wording to possibly capture both cases. It's initially why I wanted to capture two different error messages since they're technically two different reasons why we didn't generate the resulting callable/ code.

Reply:
Change it to `Not implemented: {callableName}(). REST transport is not implemented for this method yet.`